tests/unit_test/README.md

```diff
@@ -0,0 +1,100 @@
+## Introduction
+
+For XC3 we are using pytest for testing, and we have provisioned our infrastructure using Terraform. Additionally, we are using the Boto3 client library to interact with AWS services programmatically.
+
+In the following sections, we will explore how to use these tools to test our application effectively and efficiently. We will cover topics such as defining test cases, running tests, and analyzing test results. We will also discuss leveraging Terraform to create and manage the test environment and how Boto3 can help us interact with AWS services during testing.
+
+By the end of this guide, you should understand how to use pytest, Terraform, Boto3 and mock to test your application, and how these tools can help you achieve better software quality and reliability.
+
+## How will the tests be run?
+
+Tests will be run frequently throughout the development cycle, including during the build and deployment stages. It's a good practice to run tests after each code change and before each deployment to ensure that the changes don't introduce any new bugs or regressions. We have automated this process using GitHub Actions, which allows us to set up a workflow that automatically tests our code whenever changes are pushed to the repository. With GitHub Actions, you can define custom workflows in YAML files that specify the steps to run and the conditions under which they should be run. In our case, we have defined a workflow that runs the pytest command to execute the test suite whenever changes are pushed to the repository. The results of the test run are available in the GitHub Actions console, where developers can view logs and other diagnostic information.
+
+## How to run tests locally?
+First, clone the repository [XC3](https://github.com/XgridInc/xc3.git)
+and follow the [installation guide](https://docs.google.com/document/d/1YoaWoDSOjdOY2Wk3FXYHOni4wERCU7_WvsQamYkqS_o/edit) until you reach the 'terraform init' step within the 'XC3/infrastructure' directory. Once you've completed those initial steps, navigate to the 'XC3/test/unittests' directory.
+
+To set up the required dependencies, create a file named `requirements.txt` and paste the following content into the file:
+
+```plaintext
```

Comment:
I think you need to move this library list to requirementa.txt

Reply:
got it.